src/carnot/exec/otel_export_sink_node.cc

```diff
@@ -49,6 +49,8 @@ const int64_t kOTelTraceIDLength = 16;
 
 const int64_t kB3ShortTraceIDLength = 8;
 
+const int64_t otelClientTimeoutS = 5;
```

Comment:
nit: just make this `std::chrono::seconds` and then you can get rid of the `S` in the name

Reply:
Also do you have an idea of how long these requests usually take?

Reply:
I tried timing it on a cluster with our demo apps + load_gens running, and also tried running a longer query (send HTTP spans from the last 10 mins), and it was within the .3-.7 seconds range.